core/src/autogluon/core/models/ensemble/fold_fitting_strategy.py

```diff
@@ -661,9 +679,10 @@ def _update_bagged_ensemble_child_resources(self):
     def _run_parallel(self, X, y, X_pseudo, y_pseudo, model_base_ref, time_limit_fold, head_node_id):
         job_refs = []
         job_fold_map = {}
+        gpu_assignments = {}
```

Comment:
Why are we passing an empty dictionary?
Why not have `gpu_assignments` be a variable of the class: `self.gpu_assignments`?